api/Navigator.json

```diff
@@ -2506,7 +2506,8 @@
               "version_added": false
             },
             "firefox_android": {
-              "version_added": false
+              "version_added": "79",
+              "notes": "Firefox Mobile displays custom share dialog (instead of the platform-native dialog indicated by the specification)."
```

Comment:
AFAIK it's using Android feature to customize the UI, which should be still considered as "native". Chrome also shows customized UI and also several non-browser apps e.g. OneDrive.

A quick search shows how. https://stackoverflow.com/q/32478266

Reply:
I don't see a spec requirement to be "native" BTW. The spec says the user agent "can" use the OS share system, not "must". https://w3c.github.io/web-share/#examples-of-share-targets

Reply:
Fair enough. In the abstract it even says "The available share targets are not specified here; they are provided by the user agent.",

I will remove that.